packages/cli/lib/src/workflows/install_version.workflow.dart

```diff
@@ -1,25 +1,38 @@
 import 'package:fvm/exceptions.dart';
 import 'package:fvm/fvm.dart';
 import 'package:fvm/src/flutter_tools/git_tools.dart';
+import 'package:fvm/src/utils/confirm.dart';
 import 'package:fvm/src/utils/logger.dart';
 
 import '../utils/logger.dart';
 
-Future<void> installWorkflow(String version) async {
+Future<void> installWorkflow(
+  String version, {
+  bool skipConfirmation = false,
+}) async {
   try {
     assert(version != null);
 
     // If it's installed correctly just return and use cached
-    if (await LocalVersionRepo.isInstalled(version)) {
-      FvmLogger.fine('Version: $version - already installed.');
+    final isVersionInstalled = await LocalVersionRepo.isInstalled(version);
+
+    // Ensure the config link and symlink are updated
+    final project = await FlutterProjectRepo.findAncestor();
+    await FlutterProjectRepo.updateSdkLink(project);
```

Comment:
Does this work when the version does not exist?

Reply:
I think this should only be done when the version is pinned. Might need a flag to indicate this. When it is not pinned it should not automatically link as the `use` command is intended to do this.

Reply:
The previous implementation only did this when the version is pinned. The issue is that the version pinned changed outside of FVM. Essentially we are ensuring that the version in the config always matches the link. 

It checks if the project config does exist to recreate the link, and it goes through the normal install workflow which asks if you want to install the version, or returns if the verison is already installed